sentry-android-core/src/main/java/io/sentry/android/core/util/ConnectivityChecker.java

```diff
@@ -23,12 +24,18 @@ private ConnectivityChecker() {}
    *
    * @return true if the application has internet access
    */
-  public static @Nullable Boolean isConnected(@NotNull Context context, @NotNull ILogger logger) {
-    ConnectivityManager connectivityManager = getConnectivityManager(context, logger);
+  public static @Nullable Boolean isConnected(
```

Comment:
`m`: I am not a huge fan of using `null` as a return value. What about creating an enum for the result here. We have many different outcomes here: connected, not connected, min sdk not high enough, something went wrong. We could combine it too three different outcomes: `connected`, `notConnected` and `unknown`.

Reply:
min sdk is only for `getConnectionType` not for `isConnected`.
so its either connected, not connected, or unknown.
I'd prefer to also use `enums` all the type, but all of those types would need to be public.